core/src/main/java/org/springframework/security/authentication/AuthenticationTrustResolver.java

```diff
@@ -53,4 +53,17 @@ public interface AuthenticationTrustResolver {
 	 */
 	boolean isRememberMe(Authentication authentication);
 
+	/**
+	 * Indicates whether the passed <code>Authentication</code> token represents an anonymous user that
+	 * has been remembered (i.e. user that has been fully authenticated). This is a composition of 
```

Comment:
I don't think that this is true. The logic confirms that this user is *not* anonymous and is *not* remember me. Can we please adjust the wording?

Maybe:

```
Indicates whether the passed {@code Authentication} token represents a fully authenticated user (that is, neither anonymous or remember-me).
```

Reply:
updated the comment